recipes/outrigger/meta.yaml

```diff
@@ -0,0 +1,88 @@
+package:
+  name: outrigger
+  version: "1.0.0"
+
+source:
+  fn: outrigger-1.0.0.tar.gz
+  url: https://pypi.python.org/packages/13/b9/02d6db3140448685a9651918e3c2831401983d8123dd2560320283abddcc/outrigger-1.0.0.tar.gz
+  md5: 6b7c5a8ae752f79c49e2e124e609a099
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - outrigger = outrigger:main
+    #
+    # Would create an entry point called outrigger that calls outrigger.main()
+
+    - outrigger = outrigger.commandline:main
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
```

Comment:
It looks like a number of your requirements are missing from either the `conda-forge` or `defaults` channels. You will need to submit recipes for the missing ones before this recipe will build.